labgrid/driver/fastbootdriver.py

```diff
@@ -59,6 +59,11 @@ def flash(self, partition, filename):
         check_file(filename, command_prefix=self.fastboot.command_prefix)
         self("flash", partition, filename)
 
+    @Driver.check_active
+    @step(args=['cmd'])
+    def run(self, cmd):
+        self("oem exec -- ", cmd)
```

Comment:
The fixed string should be split into individual arguments, otherwise it will likely fail when used locally instead of via ssh.